src/inventory.gql

```diff
@@ -16,12 +16,44 @@ type ProductItem {
 """A simple UUID parser"""
 scalar UUID
 
+type PaginatedProductItem {
+  nodes: [ProductItem!]
+  totalCount: Int!
+  hasNextPage: Boolean!
+}
+
 type Query {
-  productItems: [ProductItem!]!
+  productItems(findProductItemsInput: FindProductItemArgs!): PaginatedProductItem!
   countProductItems(productVersionId: UUID!): Int!
   productItem(_id: UUID!): ProductItem!
```

Comment:
the parameter should be called id, not _id

Reply:
It should be Id right? I want to change all __id_ to _Id_ in a different pull request

Reply:
ok